cmd/zoekt-sourcegraph-indexserver/sg.go

```diff
@@ -166,34 +171,20 @@ func (s *sourcegraphClient) GetRepoRank(ctx context.Context, repoName string) ([
 
 // GetDocumentRanks asks Sourcegraph for a mapping of file paths to rank
 // vectors.
-func (s *sourcegraphClient) GetDocumentRanks(ctx context.Context, repoName string) (map[string][]float64, error) {
+func (s *sourcegraphClient) GetDocumentRanks(ctx context.Context, repoName string) (RepoPathRanks, error) {
```

Comment:
minor: it looks like we just treat this as an opaque value now. Should we just return a `[]byte` here and leave it upto zoekt-git-index to know how to unmarshal it?

Reply:
I found it convenient to have a struct because we also have test versions of this (like `sourcegraphFake`). So I'll keep it if you don't mind! 

I'm also tracking a list of clean-up items and this set of APIs is on it :)